web/beacon-app/index.html

```diff
@@ -2,9 +2,32 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>Ensign by Rotational Labs</title>
+    <meta name="description" content="Ensign by Rotational Labs"/>
+
+    <link rel="apple-touch-icon" href="/apple-touch-icon.png" />
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+    <link rel="manifest" href="/manifest.json" />
+    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
+   
+    <meta name="msapplication-TileColor" content="#da532c">
+    <meta name="theme-color" content="#ffffff" />
+
+    <meta property="og:url" content="https://rotational.app">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="Ensign by Rotational Labs">
+    <meta property="og:description" content="Ensign by Rotational Labs">
+    <meta property="og:image" content="">
```

Comment:
I'll create a follow up story to add images to the meta tags. Also, Twitter is listed separately because last I checked their site has stopped recognizing og tags.